packages/angular/build/src/utils/i18n-options.ts

```diff
@@ -64,6 +65,16 @@ function ensureString(value: unknown, name: string): asserts value is string {
   }
 }
 
+function ensureValidateUrlSegment(value: unknown, name: string): asserts value is string {
+  ensureString(value, name);
+
+  if (!/^[\w-]*$/.test(value)) {
+    throw new Error(
+      `Project ${name} field is malformed. Expected to match pattern: '/^[\\w-]*$/'.`,
```

Comment:
Suggestion: Can we describe this verbally rather than just referencing a regex? Newer developers aren't necessarily going to be familiar with regex or understand what pattern this is trying to express.